nimx/view_event_handling_new.nim

```diff
@@ -129,7 +129,7 @@ proc processTouchEvent*(v: View, e : var Event): bool =
                         result = s.processTouchEvent(e)
                         if result:
                             v.touchTarget = s
-                            e.setTouchTarget(v)
+                            e.setTouchTarget(s)
```

Comment:
Should not this line be just removed because touchtarget is already set in `s.processTouchEvent(e)`?